### ISSUE TITLE
[IndexBundle] Menu item not showing up

### ISSUE THREAD

**Issue by aarongerig** [coreshop/CoreShop#1809]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | yes

When installing only the Index Bundle with `composer require coreshop/index-bundle:^3.0` some issues arise in the Back-End.

**First issue: Menu item is not showing**
After the bundle is enabled, there should be a new menu item on the left. In my browser console the following two errors appear:
```log
GET http://localhost:2000/admin/coreshop/coreshop.main/menu.js net::ERR_ABORTED 500 (Internal Server Error)
```
```log
TypeError: Cannot read properties of undefined (reading 'coreshop')
```

When opening the path to the `menu.js` file directly, the following error appears:
```log
An exception has been thrown during the rendering of a template ("The menu "coreshop.main" is not defined.").
in vendor/coreshop/menu-bundle/Resources/views/menu.js.twig (line 17)
```
So my guess would be, that the menu `coreshop.main` menu is not properly defined and therefore cannot be rendered.

<s>**Second issue: NullOutput doesn't have fetch method**
When opening the Bundles & Bricks menu item, the following error message appears:
```log
Attempted to call an undefined method named "fetch" of class "Symfony\Component\Console\Output\NullOutput"
in /var/www/html/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/ExtensionManager/ExtensionManagerController.php (line 496)
```

Here Pimcore tries to [call the `fetch` method](https://github.com/pimcore/pimcore/blob/10.x/bundles/AdminBundle/Controller/ExtensionManager/ExtensionManagerController.php#L495) on the output object returned by the bundle's installer. The Index Bundle [returns a `NullOutput`](https://github.com/coreshop/CoreShop/blob/master/src/CoreShop/Bundle/IndexBundle/Installer.php#L78), which doesn't implement the `fetch` method and therefore fails.</s>

**Comment by jdreesen** [comment]
The second issue has already been fixed in Pimcore (pimcore/pimcore#10908) and should be released on Wednesday.

**Comment by aarongerig** [comment]
Resolved by https://github.com/coreshop/CoreShop/pull/1815
